projects/aca-content/src/lib/components/sidenav/components/sidenav-header.component.html

```diff
@@ -4,10 +4,9 @@
       tabindex=0
       (click)="toggleNavBar.emit()"
       (keypress)="toggleNavBar.emit()">
-      <img
-        src="{{ logo$ | async }}"
+        <adf-icon value="adf:side_nav_collapse"
         title="{{'APP.TOOLTIPS.COLLAPSE_NAVIGATION' | translate}}"
-        alt="{{ 'CORE.HEADER.LOGO_ARIA' | translate }}" />
```

Comment:
if you are removing an attribute with i18n key, you should check if it is no longer used anywhere, and remove the key from the `en.json` file too

Reply:
It is being used elsewhere